sql/modules/Reconciliation.sql

```diff
@@ -44,6 +50,18 @@ $$Submits a reconciliation report for approval.
 in_line_ids is used to specify which report lines are cleared, finalizing the
 report.$$;
 
+CREATE OR REPLACE FUNCTION reconciliation__check_balanced(in_report_id int
+      ) RETURNS NUMERIC AS
+$$
+    SELECT COALESCE((
+          SELECT SUM(crl.our_balance) - cr.their_total
```

Comment:
Does a `0` return from this function mean "balanced"? If so, do you mean to return "Yes, it's balanced" when `SUM(crl.our_balance)` is `NULL` (thus leading the difference to be `NULL`)?